grakn-kb/src/main/java/ai/grakn/kb/internal/GraknTxAbstract.java

```diff
@@ -109,6 +111,7 @@
 public abstract class GraknTxAbstract<G extends Graph> implements GraknTx, GraknAdmin {
     final Logger LOG = LoggerFactory.getLogger(GraknTxAbstract.class);
     private static final String QUERY_BUILDER_CLASS_NAME = "ai.grakn.graql.internal.query.QueryBuilderImpl";
```

Comment:
can we do `QueryBuilderImpl.getClass().getName()` instead

Reply:
We can't, because `grakn-kb` does not depend on `grakn-graql` :(

Ideally in the future, we can move `QueryBuilderImpl` and `TinkerQueryRunner` into `grakn-kb` and then we won't need this nonsense.